### PR TITLE
Fix UI

### PR DESCRIPTION
**Pull request by ana2201**
## Description
Create forum, forum and topic view fixed. Some details for dark theme, metaplex, daa and solana

## Screenshots
preview ---> https://forum-staging-ml7np0oit-usedispatch.vercel.app/
the "powered by dispatch" logo will look wrong on the landing page because the preview uses `main` on dispatch.forum, and needs to use https://github.com/usedispatch/www.dispatch.forum/pull/102 for allt he changes to look as expected

Selection_238
Selection_236
Selection_230
Selection_237

### CODE REVIEW

File: src/styles/customizations/DegenApeAcademy.css [hunk 1]
```diff
@@ -411,6 +417,21 @@
...
+
+      &.inFlight {
+        opacity: 0.75;
+        cursor: progress;
+        background: #bdddf445;
+      }
+    }
+
+    .searchBarWrapper {
+      color: black !important;
```
Comment (diegorosolino) [inline src/styles/customizations/DegenApeAcademy.css:429]: I prefer not to use `!important`, maybe we can look for another way to apply these styles preferably in other way

File: src/styles/ManageOwners.css [hunk 2]
```diff
@@ -165,6 +165,10 @@
 
       &.add {
         height: 42px;
+        background: transparent;
+        color: #454196;
```
Comment (diegorosolino) [inline src/styles/ManageOwners.css:169]: Maybe we can define `colors.css` that defines all colors used in the app and import from there into the other CSS files. I mean centralize all colors so is easier to change colors in the future, what do you think?

File: src/styles/customizations/DegenApeAcademy.css [hunk 3]
```diff
@@ -411,6 +417,21 @@
       color: white;
       background: #1d1c1c;
       font-family: 'Raleway', 'sans-serif';
```
Comment (zfaizal2) [inline src/styles/customizations/DegenApeAcademy.css:419]: fonts are still broken on this page, not sure if this has anything to do with it

**Review by zfaizal2 — DISMISSED** [review]

File: src/styles/customizations/DegenApeAcademy.css (hunk 1 above)
Comment (ana2201) [reply src/styles/customizations/DegenApeAcademy.css:429]: The search bar is a third party component, I couldn't override the styles in other way

File: src/styles/ManageOwners.css (hunk 2 above)
Comment (ana2201) [reply src/styles/ManageOwners.css:169]: I agree, but that task would take more time an effort, it should have its own PR

File: src/styles/customizations/DegenApeAcademy.css (hunk 3 above)
Comment (ana2201) [reply src/styles/customizations/DegenApeAcademy.css]: I removed that, and updated the preview link

**Review by diegorosolino — APPROVED** [review]

**Pull request merged**